lib/hexpm/accounts/web_auth.ex

```diff
@@ -0,0 +1,117 @@
+defmodule Hexpm.Accounts.WebAuth do
+  use Hexpm.Context
+
+  @moduledoc false
+
+  # A pool for storing and validating web auth requests.
+
+  alias Hexpm.Accounts.WebAuthRequest, as: Req
```

Comment:
nitpick: let's call it by its full name? (not least because using Req name will trip me every single time because https://hex.pm/packages/req :)) 

```suggestion
  alias Hexpm.Accounts.WebAuthRequest
```

Reply:
Sure.